### DISCUSSION TITLE
How to make more targets available

### DISCUSSION THREAD

**Discussion by mixedCase** [srid/rust-nix-template#25] [Q&A] [answered]
I'm trying to build a Rust project that uses the wasm32-unknown-unknown target.

How can I make it available using this template?

**Comment by srid** [comment, marked as answer]
I don't know, but see https://github.com/NixOS/nixpkgs/issues/89426

You can switch to using crane, I've been building wasm apps with it. Here's an example: https://github.com/srid/leptos-fullstack/blob/master/nix/flake-module.nix

**Reply by mixedCase** [reply]
Thank you, I'll have to change approaches then!

**Comment by srid** [comment]
This template uses `crane` now (via https://github.com/juspay/rust-flake which uses the oxalica overlay), and targets can be specified in the toolchain TOML:

https://github.com/srid/rust-nix-template/blob/253aa328ce18660fd87d0e155b93803dcd3b724b/rust-toolchain.toml#L1-L2

Example,

https://github.com/srid/leptos-fullstack/blob/4203055c17476616bec375cd7e471e91f70d26d0/rust-toolchain.toml#L3
